storage/store/collections.go

```diff
@@ -205,3 +211,33 @@ func (c *Collections) LightByTransactionID(txID flow.Identifier) (*flow.LightCol
 
 	return &collection, nil
 }
+
+// BatchStoreLightAndIndexByTransaction stores multiple light collections and indexes them by transaction ID
+// within a single batch operation.
+// No errors are expected during normal operation.
+func (c *Collections) BatchStoreLightAndIndexByTransaction(collections []flow.LightCollection, batch storage.ReaderBatchWriter) error {
```

Comment:
similar comment here about matching the prototype for `StoreLightAndIndexByTransaction`

```suggestion
func (c *Collections) BatchStoreLightAndIndexByTransaction(collection *flow.LightCollection, batch storage.ReaderBatchWriter) error {
```